.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+ignore = E203, E266, E501, W503, F403, F401, F821
+max-line-length = 79
```

Comment:
Should we use max-line-length of 119 instead? I know PEP8 says 79, but it always feels so short... @ggalves what's your opinion on this?

Reply:
yeah, we usually use 120 rather than 80. Can you change it @LeonardoFurtado ?

Reply:
Yep!

Reply:
Done!